### ISSUE TITLE
Bundled version of jQuery SVG incompatible with jQuery 1.7.2 and higher

### ISSUE THREAD

**Issue by charman** [nlplab/brat#1109]
TL;DR recap - Use this fork of jQuery SVG - https://github.com/twitwi/svg - for compatibility with jQuery 1.8+.

BRAT comes bundled with v1.4.3 of jQuery SVG (https://github.com/kbwood/svg). jQuery SVG v1.4.3 is compatible with the jQuery v1.7.1 that is bundled with BRAT, but is not compatible with jQuery 1.7.2. The last official release of jQuery SVG was v1.4.5, which introduced support for jQuery 1.7 and was released on 2012-04-28. Version v1.4.5 of jQuery SVG works with both jQuery v1.7.1 and v1.7.2.

jQuery SVG uses some internal jQuery APIs to implement custom filter selectors, and these internal APIs changed between jQuery 1.7 and jQuery 1.8:

http://stackoverflow.com/questions/11624345/getting-the-match-object-in-a-custom-filter-selector-in-jquery-1-8

The https://github.com/twitwi/svg fork of jQuery SVG updates the package for compatibility with jQuery v1.8 and v1.9 (and, in my tests, jQuery 1.11), but is not compatible with jQuery 1.7. This fork seems to be the only active fork of the project, with four contributors so far.
